.rvmrc

```diff
@@ -1 +0,0 @@
-rvm use 2.3.1@localsupport --create
```

Comment:
Oh, I'm sorry, I did not delete it on purpose. Perhaps, it was deleted accidentally and I did not notice, so did not create again. I'll fix it.